src/main/scala/firrtl/ExecutionOptionsManager.scala

```diff
@@ -260,6 +262,7 @@ extends ComposableOptions {
   def getEmitterAnnos(optionsManager: ExecutionOptionsManager): Seq[Annotation] = {
     // TODO should this be a public function?
     val emitter = compilerName match {
+      case "none" => classOf[HighFirrtlEmitter]
```

Comment:
Is `HighFirrtlEmitter` the same as a "Chirrtl emitter"?

Reply:
Other than the returned CircuitState type, yes. I guess I should make a ChirrtlEmitter and have the none compiler use that

Reply:
Is an emitter of a higher form "compatible" with lower forms? So if I use the none compiler and pass in a low FIRRTL input, will the output still be low FIRRTL even if I use the ChirrtlEmitter?

Reply:
Theoretically, that's the idea (lower forms are subsets of higher forms). Practically, all the FIRRTL emitters are the exact same.

Reply:
It's "compatible" in that all FIRRTL emitters do the same thing, but the output CircuitState will always have the form of the emitter itself. In fact, the output form of any transform (or emitter) is always statically known and independent of the form of the input CircuitState. This could possibly be changed but making it statically known makes it possible to statically schedule transforms which has some utility. 

You can see the actual implementations of the various Firrtl Emitters here: https://github.com/freechipsproject/firrtl/blob/b90589f5cd9d4048ada2a05d5225874791546170/src/main/scala/firrtl/Emitter.scala#L115